src/terraform/install.sh

```diff
@@ -27,7 +27,7 @@ TFSEC_SHA256="${TFSEC_SHA256:-"automatic"}"
 TERRAFORM_DOCS_SHA256="${TERRAFORM_DOCS_SHA256:-"automatic"}"
```

Comment:
Can we bump the patch version in here? thanks!

https://github.com/devcontainers/features/blob/79a042923039bb6c974a78ffed9db264ff5e4d57/src/terraform/devcontainer-feature.json#L3

Reply:
Thanks. Bumped to 1.3.3.